inspirehep/modules/workflows/workflows/article.py

```diff
@@ -349,12 +349,10 @@
                         [
                             reject_record('Article automatically rejected'),
```

Comment:
this is incorrect, you reach this also if it's manually rejected and not for a hidden collection.

Reply:
Do we need an exact reason here or can I just put "Article rejected"?

Reply:
no, that's fine